man/systemd-run.xml

```diff
@@ -394,6 +394,28 @@ Dec 08 20:44:48 container systemd[1]: Started /bin/touch /tmp/foo.</programlisti
 
       <programlisting># systemd-run -t --send-sighup /bin/bash</programlisting>
     </example>
+
+    <example>
+      <title>Start <command>screen</command> as a user service</title>
+
+      <programlisting>$ systemd-run --scope --user screen -d -m
+Running scope as unit run-r14b0047ab6df45bfb45e7786cc839e76.scope.
+
+$ screen -ls
+There is a screen on:
+        492..laptop     (Detached)
+1 Socket in /var/run/screen/S-fatima.
+</programlisting>
+
+      <para>To allow the screen to persist after the user logs out,
+      even if the session scope is terminated, enable lingering:</para>
+
```

Comment:
If lingering is disabled, `user@.service` will be killed when the user logs out from the last session. What about the following wording:

By default, the screen which is now running as a scope unit underneath the user's systemd instance will be terminated when the user logs out of their *last* session. To prevent that and allow <command>screen</command> to persist, enable lingering: